interop-testing/src/test/java/io/grpc/testing/integration/AutoWindowSizingOnTest.java

```diff
@@ -31,8 +30,11 @@
 
   @BeforeClass
   public static void turnOnAutoWindow() {
-    InternalHandlerSettings.enable(true);
-    InternalHandlerSettings.autoWindowOn(true);
+    // TODO(carl-mastrangelo): Reenable this when https://github.com/grpc/grpc-java/issues/3856 is
```

Comment:
This bothers me, as appearance is quite different from actuality. Could we do an Assume.assumeTrue(false) instead?